doc/ikfast/ikfast_tutorial.rst

```diff
@@ -73,210 +53,79 @@ Inside your catkin workspace's ``./src`` directory: ::
-
-The speed and success of this process will depend on the complexity of your robot. A typical 6 DOF manipulator with 3 intersecting axis at the base or wrist will take only a few minutes to generate the IK.
-
-**Known issue**
-There is a known bug that causes the ``--freeindex`` argument to handle tree indexes incorrectly. Say ``--baselink=2``, ``--eelink=16`` and the links indexed from 3 to 9 are not related to the planning group chain. In that case ``--freeindex`` will correctly interpret index 2 as link 2, but incorrectly interpret index 3 as link 10, index 4 as link 11, ... and index 9 as link 16.
-
-You should consult the OpenRAVE mailing list and ROS Answers for information about 5 and 7 DOF manipulators.
-
-Create Plugin
--------------
-
-Create the package that will contain the IK plugin. We recommend you name the package ``"$MYROBOT_NAME"_ikfast_"$PLANNING_GROUP"_plugin``: ::
-
- cd ~/catkin_ws/src
- export MOVEIT_IK_PLUGIN_PKG="$MYROBOT_NAME"_ikfast_"$PLANNING_GROUP"_plugin
- catkin_create_pkg "$MOVEIT_IK_PLUGIN_PKG"
-
-Build your workspace so the new package is detected (can be 'roscd'): ::
-
- catkin build
-
-Create the plugin source code. Run ``rosrun moveit_kinematics create_ikfast_moveit_plugin.py -h`` to see optional arguements: ::
-
- rosrun moveit_kinematics create_ikfast_moveit_plugin.py "$MYROBOT_NAME" "$PLANNING_GROUP" "$MOVEIT_IK_PLUGIN_PKG" "$BASE_LINK_NAME" "$EEF_LINK_NAME" "$IKFAST_OUTPUT_PATH"
-
-For the Panda we need to use some of the optional parameters: ::
-
- rosrun moveit_kinematics create_ikfast_moveit_plugin.py --moveit_config_pkg="panda_moveit_config" --robot_name_in_srdf="panda" "$MYROBOT_NAME" "$PLANNING_GROUP" "$MOVEIT_IK_PLUGIN_PKG" "$BASE_LINK_NAME" "$EEF_LINK_NAME" "$IKFAST_OUTPUT_PATH"
-
-
-Or without ROS: ::
-
- python /path/to/create_ikfast_moveit_plugin.py "$MYROBOT_NAME" "$PLANNING_GROUP" "$MOVEIT_IK_PLUGIN_PKG" "$BASE_LINK_NAME" "$EEF_LINK_NAME" "$IKFAST_OUTPUT_PATH"
+  catkin build
 
 Usage
 -----
-The IKFast plugin should function identically to the default KDL IK Solver, but with greatly increased performance. The MoveIt configuration file is automatically edited by the moveit_ikfast script but you can switch between the KDL and IKFast solvers using the *kinematics_solver* parameter in the robot's kinematics.yaml file: ::
+The IKFast plugin can be used as a drop-in replacement for the default KDL IK Solver, but with greatly increased performance. The MoveIt configuration file should be automatically edited by the generator script but in some cases this might fail. In this situation you can switch between the KDL and IKFast solvers using the *kinematics_solver* parameter in the robot's kinematics.yaml file: ::
 
- rosed "$MYROBOT_NAME"_moveit_config kinematics.yaml
+  rosed "$MYROBOT_NAME"_moveit_config kinematics.yaml
```

Comment:
Unless I am missing something I don't think this needs the space.

Reply:
It does in order to be formatted as a command.